samza-core/src/main/java/org/apache/samza/zk/ZkUtils.java

```diff
@@ -163,7 +183,12 @@ String readProcessorData(String fullPath) {
         String fullPath = String.format("%s/%s", processorPath, child);
         processorIds.add(new ProcessorData(readProcessorData(fullPath)).getProcessorId());
       }
-
+      Collections.sort(processorIds, new Comparator<String>() {
```

Comment:
Just curious, why're we sorting the processorIds in `getActiveProcessorsIDs` api.

We already have `getSortedActiveProcessorsIDs` which does the same(or does it make sense to merge them both).